src/assets/services_lightingsoundpower.json

```diff
@@ -434,6 +434,26 @@
 						"description": "Heavy duty power distribution for your large equipment such as food warmers or power to many tables for attendees. Not necessary for small electronics that just need an extension cord."
 					}
 				]
+			},
+			{
+				"title": "Inflatable Screen",
+				"isBoolean": true,
+				"choices": [
+					{
+						"title": "Inflatable Screen",
+						"id": "IS",
+						"price": 0,
+						"imageUrl": "assets/images/inflatable_screen.jpg",
+						"description": "Large outdoor inflatable screen commonly used on the Quad with our Digital Projector (Christie CP2210).",
```

Comment:
The Christie CP2210 is not used on the quad.

Reply:
Fixed in [781103e](https://github.com/WPI-LNL/workorderwizard/pull/17/commits/781103eff92ce13a15f0d6236d4bf47f02d95b9b)